### ISSUE TITLE
Noauth and plexurl options

### ISSUE THREAD

**Issue by brettg0396** [synclounge/synclounge#304]
I host plex on a domain with a custom access url plex.domain.com. I host synclounge the same way on something like domain.com/slweb. They are hosted on the same machine and recently I had an idea.

In Plex, I believe there is an option to circumvent authorization from certain IPs, say, on the same LAN or subnet. I already have the synclounge app set up behind authorization with a rotating password known to me. What if I could tell synclounge to use access my plex media server directly (instead of through plex.tv as I'm sure it does) and thereby require no login? Users could pick a nickname and be instanced as local connections essentially proxied through the synclounge app running on the same machine. With this, I could host large watch parties by giving out the rotating pass for that day and nobody would need to have an actual plex account.

It's an odd request for a specific edge case, but I'm sure there are those who self-host synclounge who might appreciate the option if it weren't too much trouble.

**Comment by ttshivers** [comment]
You are correct that SyncLounge currently queries plex.tv for servers and clients: https://github.com/synclounge/synclounge/blob/e720eebe549855c8ea862b6a334340de4e810c61/src/store/modules/plex/actions.js#L58

Adding noauth support like that would take some rewriting of SyncLounge. The auth logic would need to be disabled in that situation. A way to manually input servers / clients through the UI or config would need to be added.

Something that could potentially offer a similar result with the way SyncLounge is currently is creating a bot plex user and hardcoding a token in your build of SyncLounge and restricting that behind your external authorization you were talking about.
